content/en/docs/instrumentation/js/manual.md

```diff
@@ -250,73 +483,330 @@ const opentelemetry = require("@opentelemetry/api");
+  for (let i = 0; i < rolls; i++) {
+    result.push(rollOnce(min, max));
+  }
+  return result
+}
+
+module.exports = { rollTheDice }
+{{< /tab >}}
+
+{{< /tabpane>}}
+
+<!-- prettier-ignore-end -->
+
+### Create spans
+
+Now that you have [tracers](/docs/concepts/signals/traces/#tracer) initialized,
+you can create [spans](/docs/concepts/signals/traces/#spans).
+
+The code below illustrates how to create an active span, which is the most
+common kind of span.
+
+<!-- prettier-ignore-start -->
+
+{{< tabpane langEqualsHeader=true >}}
+
+{{< tab TypeScript >}}
+import { trace, Span } from "@opentelemetry/api";
+
+/* ... */
+
+export function rollTheDice(rolls: number, min: number, max: number) {
+  // Create a span. A span must be closed.
+  return tracer.startActiveSpan('rollTheDice', (span: Span) => {
+    const result: number[] = []
+    for(let i = 0; i < rolls; i++) {
+      result.push(rollOnce(min, max));
+    }
+    // Be sure to end the span!
+    span.end();
+    return result
+  })
+}
+{{< /tab >}}
+
+{{< tab JavaScript >}}
+function rollTheDice(rolls, min, max) {
+  // Create a span. A span must be closed.
+  return tracer.startActiveSpan('rollTheDice', (span) => {
+    const result = []
+    for (let i = 0; i < rolls; i++) {
+      result.push(rollOnce(min, max));
+    }
+    // Be sure to end the span!
+    span.end();
+    return result
+  })
+}
+{{< /tab >}}
+
+{{< /tabpane>}}
+
+<!-- prettier-ignore-end -->
+
+If you followed the instructions using the [example app](#example-app) up to
+this point, you can copy the code above in your library file `dice.ts` (or
+`dice.js`). You should now be able to see spans emitted from your app.
+
+Start your app as follows, and then send it requests by visiting
+<http://localhost:8080/rolldice?rolls=12> with your browser or `curl`.
+
+<!-- prettier-ignore-start -->
+{{< tabpane lang=shell persistLang=false >}}
+
+{{< tab TypeScript >}}
+ts-node --require ./instrumentation.ts app.ts
+{{< /tab >}}
+
+{{< tab JavaScript >}}
+node --require ./instrumentation.js app.js
+{{< /tab >}}
+
+{{< /tabpane >}}
+<!-- prettier-ignore-end -->
+
+After a while, you should see the spans printed in the console by the
+`ConsoleSpanExporter`, something like this:
+
+```json
+{
+  "traceId": "6cc927a05e7f573e63f806a2e9bb7da8",
+  "parentId": undefined,
+  "name": "rollTheDice",
+  "id": "117d98e8add5dc80",
+  "kind": 0,
+  "timestamp": 1688386291908349,
+  "duration": 501,
+  "attributes": {},
+  "status": { "code": 0 },
+  "events": [],
+  "links": []
+}
+```
 
 ### Create nested spans
 
 Nested [spans](/docs/concepts/signals/traces/#spans) let you track work that's
-nested in nature. For example, the `doWork` function below represents a nested
-operation. The following sample creates a nested span that tracks the `doWork`
-function:
+nested in nature. For example, the `rollOnce()` function below represents a
+nested operation. The following sample creates a nested span that tracks
+`rollOnce()`:
```

Comment:
Recommend quickly elaborating that in JS/TS, you need to use `startActiveSpan` here, otherwise you'll have to use the context API to explicitly set spans as active in the current context.

...which is a pain, hence the API! ...but that can be left out :)

Reply:
I added a comment about that a little bit up in the "Create Spans" section:

https://github.com/open-telemetry/opentelemetry.io/pull/2965/files#diff-d5f10bf675fa9b56597823729a28aef6cddf15319e74e6518a546c09297d9021R635